tests/test_resume.py

```diff
@@ -24,9 +26,14 @@ def test_resume(tmp_dir, resume, steps, metrics):
     for new_metric in [0.7, 0.6]:
         dvclive.log_metric("metric", new_metric)
         dvclive.next_step()
+    dvclive.end()
```

Comment:
QQ: is this one actually needed or it's just for a symmetry?

Reply:
Just for symmetry/best practice. I think the original year might have been from before we had and end method.